deployment/deployment-grpc-v2/02-service.yaml

```diff
@@ -4,11 +4,14 @@ metadata:
  name: contour
  namespace: heptio-contour
  annotations:
-  # This annotation puts the AWS ELB into "TCP" mode so that it does not 
-  # to http negotiation at the ELB edge. This means the remote address of 
+  # This annotation puts the AWS ELB into "TCP" mode so that it does not
+  # to http negotiation at the ELB edge. This means the remote address of
```

Comment:
s/to/do/

Reply:
Should this file be a soft link to common/services.yaml?

Reply:
It is, or that is it say, it should be. I'll double check.